compiler/src/iree/compiler/Codegen/LLVMGPU/ConvertToROCDL.cpp

```diff
@@ -152,6 +152,7 @@ struct ConvertToROCDLPass final
       arith::populateArithToAMDGPUConversionPatterns(
           patterns, /*convertFP8Arithmetic=*/true, /*saturateFP8Truncf=*/false,
           /*allowPackedF16Rtz=*/false, /*chipset=*/*maybeChipset);
+      arith::populateCeilFloorDivExpandOpsPatterns(patterns);
```

Comment:
I wonder if we should add these patterns to `populateArithToAMDGPUConversionPatterns`

Reply:
I don't think `ArithToAMDGPu` creates any of these ops.

Where are these non-typical divides coming from?

Reply:
We are generating them in iree-turbine, here specifically: https://github.com/iree-org/iree-turbine/blob/63aaf873973aef48e829712a9e1c849cbe000406/iree/turbine/kernel/wave/codegen.py#L323